src/classes/ERR_Notifier.cls

```diff
@@ -74,11 +76,20 @@ public class ERR_Notifier {
                     for (User u : useremaillist)
                         sendList.add(u.email);
                 } else if(errorNotifRecipient == NotificationOptions.sysAdmins) {
-                    List<User> sysadminlist = [SELECT email FROM User WHERE User.Profile.Name = 'System Administrator' AND isActive = true];
-                    for (User u : sysadminlist) {
-                        sendList.add(u.email);
+                    Set<Id> systemAdminProfileIds = UTIL_Profile.getInstance().getProfileIds('System Administrator');
```

Comment:
As discussed this morning, let's extract the code in this `else if` statement into a separate method that is testable. Even though `emailNotificationRecipients` variable makes `sendErrorNotifications()` testable, it is best to remove it since it is used in tests only.

The Work Item task is to retrieve Profile Ids for a specified Profile Name in order to optimize the User query, so using `UTIL_Profile` is a proper way instead re-writing the query this time and every time Profile Ids are required.